src/earn.js

```diff
@@ -0,0 +1,102 @@
+import getWeb3 from './web3/getWeb3'
+import yDAI from './abis/yearn/yDAI.json'
+import ySUSD from './abis/yearn/ySUSD.json'
+import yTUSD from './abis/yearn/yTUSD.json'
+import yUSDC from './abis/yearn/yUSDC.json'
+import yUSDT from './abis/yearn/yUSDT.json'
+import cTokenAbi from './abis/cTokenAbi.json'
+import Data from './data.json'
```

Comment:
Perhaps we could name this file `addresses.json`

Reply:
Typically capitalized variables indicate class or object constructors. All capitalized, ex: `ADDRESSES` indicates a constant.